src/mixins/filing-template-mixin.ts

```diff
@@ -113,6 +112,14 @@ export default class FilingTemplateMixin extends DateMixin {
    * @returns the filing body to save
    */
   buildIncorporationFiling (): IncorporationFilingIF {
+    function fixNullAddressType (orgPeople: OrgPersonIF[]): OrgPersonIF[] {
+      return orgPeople.map(p => {
+        if (p.deliveryAddress?.addressType === null) delete p.deliveryAddress.addressType
+        if (p.mailingAddress?.addressType === null) delete p.mailingAddress.addressType
```

Comment:
The conditional chaining is required for the unit tests :( but I not fixing that now.